keyboards/strech/soulstone/info.json

```
@@ -0,0 +1,85 @@
+{
+    "keyboard_name": "Soulstone",
+    "maintainer": "Strech",
+    "manufacturer": "Strech",
+    "usb": {
+        "vid": "0xB030",
+        "pid": "0x0001",
+        "device_version": "1.0.0"
+    },
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "features": {
+        "bootmagic": true,
+        "mousekey": false,
```

This is preferred to be enabled by default, if possible.
```suggestion
        "mousekey": true,
```

---

If space is an issue, enabling lto is pprobably a good idea.

---

I will check the complication, thanks!